### ISSUE TITLE
Dialog Dismiss

### ISSUE THREAD

**Issue by HardikLohogaonkar** [pwittchen/RxBiometric#48]
If we tap outside Biometric Dialog it is dismissing. It should remain as it is.

**Comment by pwittchen** [comment]
Are you sure it should remain as it is? This is system dialog and I didn't change its behavior, so I suppose it's default behavior.

**Comment by HardikLohogaonkar** [comment]
Yes. If I accidentally tap outside dialog it will automatically close without authorizing, also there is a method named as onTouchOutside(). If you give us this option it will beneficial.

**Comment by pwittchen** [comment]
Ok, that sounds like a good idea.

**Comment by HardikLohogaonkar** [comment]
Just add this method to your dialog "setCanceledOnTouchOutside(false)"
This will prevent dialog from closing if we tap outside dialog.

**Comment by pwittchen** [comment]
Where exactly? [BiometricPrompt.PromptInfo](https://developer.android.com/reference/androidx/biometric/BiometricPrompt.PromptInfo) doesn't have such method.
